src/python/bot/fuzzers/options.py

```diff
@@ -171,6 +174,11 @@ def get_fuzz_target_options(fuzz_target_path):
   if not os.path.exists(options_file_path):
     return None
 
+  if not options_file_path:
```

Comment:
nit: move this to immediately after line 167-168.

Reply:
Done